### PR TITLE
Bug 1870490: roles/openshift-node: Verify host matches cluster FIPS

### PR DESCRIPTION
**Pull request by mtnbikenc**
Determines the cluster FIPS status by retrieving the rendered-worker
machineconfig and checking .spec.fips.

### CODE REVIEW

**Comment by mtnbikenc** [comment]
This is blocked on https://bugzilla.redhat.com/show_bug.cgi?id=1885576

File: roles/openshift_node/tasks/install.yml [hunk 1]
```diff
@@ -1,4 +1,42 @@
...
+  run_once: true
+  register: oc_get_machineconfigs
+  until:
+  - oc_get_machineconfigs.stdout != ''
+  changed_when: false
+
+- name: Set fact rendered_worker_machineconfig
+  set_fact:
+    rendered_worker_machineconfig: "{{ oc_get_machineconfigs.stdout | regex_search('^.*rendered-worker.*$', multiline=True)}}"
+  run_once: true
```
Comment (abhinavdahiya) [inline roles/openshift_node/tasks/install.yml:17]: we can actually find our rendered machine config by using 

```
$ oc get machineconfigpool worker --output=jsonpath='{.status.configuration.name}'
```

see the printer column for the machineconfigpool https://github.com/openshift/machine-config-operator/blob/4fe8559913b80253827df205b9e389ea309fc39c/install/0000_80_machine-config-operator_01_machineconfigpool.crd.yaml#L10-L12

Comment (mtnbikenc) [reply roles/openshift_node/tasks/install.yml]: Updated.

**Comment by mtnbikenc** [comment]
Mirror should be updated.
/retest

**Pull request merged**